app/src/main/java/ai/elimu/appstore/onboarding/DeviceRegistrationActivity.java

```diff
@@ -88,7 +88,7 @@ protected void onPostExecute(String jsonResponse) {
                         // Device was successfully registered
                         Timber.i("Device was successfully registered");
 
-                        AppPrefs.saveRegisterStatus(true);
+                        AppPrefs.saveDeviceRegisterStatus(true);
```

Comment:
Rename to match the get method: `setDeviceRegistered`

Reply:
Yeah, just pushed new commit. Kindly have a look again.